lnbits/core/services.py

```diff
@@ -402,6 +401,22 @@ class PaymentKwargs(TypedDict):
     return invoice.payment_hash
 
 
+def _check_wallet_balance(
```

Comment:
this function was extracted because the parent function was getting too large and flake8 complained